packages/orchestrator-ui-components/src/components/WfoSettings/WfoEngineStatus.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiText } from '@elastic/eui';
+
+import { useOrchestratorTheme } from '@/hooks';
+import { WfoStatusDotIcon } from '@/icons';
+import { useGetEngineStatusQuery } from '@/rtk';
+import { EngineStatus } from '@/types';
+
+export const WfoEngineStatus = () => {
+    const { theme } = useOrchestratorTheme();
+    const { data } = useGetEngineStatusQuery();
+    const { engineStatus, runningProcesses } = data || {};
+    const isRunning = engineStatus === EngineStatus.RUNNING;
+    const t = useTranslations('settings.page');
+
+    return (
+        <EuiPanel hasShadow={false} color="subdued" paddingSize="l">
+            <EuiFlexGroup direction="column" gutterSize="s">
+                <EuiFlexItem>
+                    <EuiText size="s">
+                        <h4>{t('engineStatusTitle')}</h4>
+                    </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem css={{ flexDirection: 'row' }}>
+                    <EuiText size="s" style={{ minWidth: 200 }}>
```

Comment:
Applicable on multiple places in this PR: try to avoid these magic numbers. Use a multiplication of the base value in the theme.

Reply:
This is code thats copied and was there already